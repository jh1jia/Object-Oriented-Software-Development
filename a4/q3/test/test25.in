test22.txt dropfirst 1
