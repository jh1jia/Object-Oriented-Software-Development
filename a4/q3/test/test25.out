DDDdddd
bcdefgH
023aAc3
5%dC
C
c
Dd20#dCe00
0
